File: Authorization.Web/ClientApp/src/app/models/api/org-user.ts
import { Org } from './org';
import { OrgUserRole } from './org-user-role';
import { User } from './user';

export interface OrgUser {
  id: number;
  orgId: number;
  userId: number;

  org: Org;
  user: User;

  orgUserRoles: OrgUserRole[];
}

File: Authorization.Data/Entities/OrgUser.cs
using System;
using System.Collections.Generic;

namespace Authorization.Data.Entities
{
    public class OrgUser
    {
        public int Id { get; set; }
        public int OrgId { get; set; }
        public int UserId { get; set; }
        
        public Org Org { get; set; }
        public User User { get; set; }

        public List<OrgUserRole> OrgUserRoles { get; set; }
    }
}